apps/judicial-system/web/src/routes/Shared/AppealToCourtOfAppeals/AppealCaseFiles.tsx

```diff
@@ -80,62 +80,37 @@ const AppealFiles = () => {
       : constants.SIGNED_VERDICT_OVERVIEW_ROUTE
   }/${id}`
 
-  const newUploadFiles = uploadFiles.filter((file) => {
-    return (
-      workingCase.caseFiles?.find((caseFile) => caseFile.id === file.id) ===
-      undefined
```

Comment:
The variable `uploadsFiles` from `useUploadFiles` now only contains new files so there is no need to filter the files.